src/integration/HubspotApiModel.ts

```diff
@@ -41,6 +41,39 @@ export class HubspotApiModel {
         }
     }
 
+    public async createObject(objectType:string, createContactBody:object){
+        try{
+            const createContactURL = 'https://api.hubapi.com/crm/v3/objects/' + objectType;
+            const response = await axios.post(createContactURL, createContactBody,{
+        
+                params: {
+                    hapikey: this.HAPIKEY
+                }
+            });
+            return response;
+        } catch (e) {
+            console.error("Was not able to create contact");
+            console.log(e);
```

Comment:
Perhaps adding  inside the catch return JSON.parse(e.response.body); This will return a promise rejection to HubApiCall function.